src/data/store.js

```diff
@@ -18,9 +17,6 @@ const store = createStore(
   composeWithDevTools(applyMiddleware(thunkMiddleware, sagaMiddleware, loggerMiddleware)),
 );
 
-const apiService = new UserAccountApiService(apiClient, process.env.LMS_BASE_URL);
```

Comment:
Since we don't use UserAccountApiService in the sagas now, this can all go away.